src/main/java/bussiness_layer/dto/GroupDTO.java

```diff
@@ -9,27 +9,15 @@
 @NoArgsConstructor
 public class GroupDTO implements Comparable<GroupDTO>{
```

Comment:
Use only one of the "DTO" or "dto" word through the entire app in order to be persistent and to avoid confusions.